serving/docs/lmi_new/user_guides/trt_llm_user_guide.md

```diff
@@ -97,39 +97,42 @@ print(outputs)
+| option.max_output_len	                                        | >= 0.25.0   | LMI	               | Maximum output token size you expect the model to have per request. This is a compilation parameter that set to the model for Just-in-Time compilation. If you set this value too low, the model will unable to produce tokens beyond the value you set.	                                                                                                                                                                                                                                                                                                                                            | Default: `512`	                                                                                                                                                             |
+| option.use_custom_all_reduce	                                 | >= 0.25.0   | LMI	               | Custom all reduce kernel is used for GPUs that have NVLink enabled. This can help to speed up model inference speed with better communication. Turn this on by setting true on P4D, P4De, P5 and other GPUs that are NVLink connected	                                                                                                                                                                                                                                                                                                                                                               | `true`, `false`. <br/> Default is `false`	                                                                                                                                  |
+| option.tokens_per_block	                                      | >= 0.25.0   | Pass Through	      | tokens per block to be used in paged attention algorithm	                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            | Default values is `64`	                                                                                                                                                     |
```

Comment:
I think we change the default to 128 @ydm-amazon to confirm

Reply:
Yes, the default is 128 now

Reply:
is the default of 128 true for v8?

Reply:
updated option.tokens_per_block default to 128